src/Library/demos/Actions/main.py

```diff
@@ -0,0 +1,70 @@
+import gi
+
+gi.require_version("Gtk", "4.0")
+gi.require_version("Adw", "1")
+from gi.repository import Gtk, Adw, GLib, Gio
+import workbench
+
+demo: Adw.StatusPage = workbench.builder.get_object("demo")
+
+demo_group = Gio.SimpleActionGroup()
+demo.insert_action_group("demo", demo_group)
+
+# Action with no state or parameters
+simple_action = Gio.SimpleAction(
+  name = "simple"
+)
+
+simple_action.connect("activate",
+  lambda action, _ : print(f"{action.get_name()} action activated")
+)
+
+demo_group.add_action(simple_action)
+
+# Action with parameter
+bookmarks_action = Gio.SimpleAction(
+  name = "open-bookmarks",
```

Comment:
One example for PEP 8 is that there should be no spaces around the `=` in keyword arguments.